### PR TITLE
EPRS: 7.5 release

### PR DESCRIPTION
**Pull request by ccestes**
## What Changed?

Changes part of the 7.5 release are: 

- [XDB-1885](https://enterprisedb.atlassian.net/browse/XDB-1885)
- [XDB-1889](https://enterprisedb.atlassian.net/browse/XDB-1889)
- [XDB-1888](https://enterprisedb.atlassian.net/browse/XDB-1888)

[XDB-1885]: https://enterprisedb.atlassian.net/browse/XDB-1885?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[XDB-1889]: https://enterprisedb.atlassian.net/browse/XDB-1889?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[XDB-1888]: https://enterprisedb.atlassian.net/browse/XDB-1888?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: product_docs/docs/eprs/7/02_overview/04_design_replication_system/02_design_considerations.mdx [hunk 1]
```diff
@@ -33,6 +33,20 @@ CREATE TABLE dept_uk2 (
...
 -   Views can be members of snapshot-only publications. In the subscription database, a view is replicated as a table.
+-   Materialized views can be members of snapshot-only publications from Oracle to EDB Postgres Advanced Server databases. The materialized views are replicated as tables. 
+    1. Use the `‘tables’` parameter for specifying materialized views while creating a publication via EDB Replication Server CLI. While creating a snapshot-only publication via Replication Console, the view type currently appears as a table. 
+    1. We recommend that useres create the views with auto-refresh in Oracle. If the views are not auto-refresh they are marked as invalid upon updating the base table until refreshed explicitly. EDB Replication Server currently does not support subscription creation or snapshot for invalid views.
```
Comment (anwar-sahib) [inline product_docs/docs/eprs/7/02_overview/04_design_replication_system/02_design_considerations.mdx:38]: We recommend that **users** create

File: product_docs/docs/eprs/7/02_overview/04_design_replication_system/02_design_considerations.mdx [hunk 2]
```diff
@@ -33,6 +33,20 @@ CREATE TABLE dept_uk2 (
...
+       The following error appears during a snapshot where the view is invalid:
+
+       ```sql
+       INFO: MTK-14001: One or more tables are missing from the source Oracle database.
+       ```
+
+       With this error, the user must refresh the view before taking the snapshot. 
+
+       !!! Note
+           Snapshot-only replication for views is currently tested and certified only for Oracle to EDB Postgres Advanced Server databases.
```
Comment (anwar-sahib) [inline product_docs/docs/eprs/7/02_overview/04_design_replication_system/02_design_considerations.mdx:49]: Change to Snapshot-only replication for **Materialized** views.
Views are supported for all combinations. 
Materialized views are tested and certified only for Oracle to EDB Postgres Advanced Served databases.

File: product_docs/docs/eprs/7/02_overview/04_design_replication_system/02_design_considerations.mdx (hunk 1 above)
Comment (ccestes) [reply product_docs/docs/eprs/7/02_overview/04_design_replication_system/02_design_considerations.mdx:38]: Good catch

**Review by mlewandowski-edb — APPROVED** [review]
Approved with some minor changes I made.

**Review by mlewandowski-edb — APPROVED** [review]

**Pull request merged**